config/main.go

```diff
@@ -27,4 +27,3 @@ var (
 	// If requirepass is set to an empty string, no authentication is required
 	RequirePass string = constants.EmptyStr
 )
-
```

Comment:
nit: Please fix

Reply:
This was suggested by the linter after I ran the `golangci-lint run` command. Do I still remove this edit?

Reply:
Should be good then